test/integration/ledger/node/votingRegistration.js

```diff
@@ -7,43 +7,70 @@ const { signingFiles } = require('./signingFiles')
 const { addresses } = require('./addresses')
 const { getTransport } = require('./speculos')
 
-// TODO add cip36 ones
 const votingRegistrations = {
   withTestnetBaseAddress0: {
-    network: 'TESTNET_LEGACY',
-    auxiliarySigningFiles: [signingFiles.payment0, signingFiles.stake0],
+    delegations: [
+      {
+        votePublicKey: '3b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b7',
+        voteWeight: 2,
+      },
+    ],
     hwStakeSigningFile: signingFiles.stake0,
     rewardAddressBech32: addresses.testnet.base0,
-    votePublicKeyHex: '3b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b7',
     nonce: 165564,
-    signedVotingRegistrationMetaDataHex: 'a219ef64a40158203b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b702582066610efd336e1137c525937b76511fbcf2a0e6bcf0d340a67bcb39bc870d85e80358390014c16d7f43243bd81478e68b9db53a8528fd4fb1078d58d54a7f11241d227aefa4b773149170885aadba30aab3127cc611ddbc4999def61c041a000286bc19ef65a10158408a8cca0fbc26626c65eaf28608da21527c70c8f6136e3eb5166e8d665ef4f9f55e0b8063393d63bfa0104a1ac737a10c1c41a93a68d101a4cb0b405ec5bb6707',
+    votingPurpose: 0,
+    network: 'TESTNET_LEGACY',
```

Comment:
just because i read it here, the legacy testnet will be retired in a few days. i guess its time to move to PREPROD or PREVIEW?

Reply:
Yes, though I would leave that for another PR when we finish this one and the deprecated features removal. I created an [issue](https://github.com/vacuumlabs/cardano-hw-cli/issues/145).